### PR TITLE
support set redis cluster password

### PR DESCRIPTION
**Pull request by ahern88**
Now redis cluster sink setting password is not supported

### CODE REVIEW

File: flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/RedisTableSinkFactory.java [hunk 1]
```diff
@@ -57,6 +57,7 @@
         properties.add(REDIS_COMMAND);
         properties.add(REDIS_NODES);
         properties.add(REDIS_MASTER_NAME);
+        properties.add(CLUSTER_PASSWORD);
```
Comment (lresende) [inline flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/RedisTableSinkFactory.java:60]: Could we follow the same pattern for the variables (e.g. `REDIS_CLUSTER_PASSWORD`)

File: flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/config/handler/FlinkJedisClusterConfigHandler.java [hunk 2]
```diff
@@ -16,10 +16,6 @@
  */
 package org.apache.flink.streaming.connectors.redis.common.config.handler;
 
-import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.REDIS_CLUSTER;
-import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.REDIS_MODE;
-import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.REDIS_NODES;
-
```
Comment (lresende) [inline flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/config/handler/FlinkJedisClusterConfigHandler.java:22]: Could we please continue using explicit imports?

File: flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/config/handler/FlinkJedisClusterConfigHandler.java [hunk 3]
```diff
@@ -31,6 +27,8 @@
 import org.apache.flink.streaming.connectors.redis.common.hanlder.FlinkJedisConfigHandler;
 import org.apache.flink.util.Preconditions;
 
+import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.*;
```
Comment (lresende) [inline flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/config/handler/FlinkJedisClusterConfigHandler.java:30]: see above

File: flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/descriptor/RedisValidator.java [hunk 4]
```diff
@@ -30,6 +30,7 @@
     public static final String REDIS_MASTER_NAME = "master.name";
     public static final String SENTINELS_INFO = "sentinels.info";
     public static final String SENTINELS_PASSWORD = "sentinels.password";
+    public static final String CLUSTER_PASSWORD = "cluster.password";
```
Comment (lresende) [inline flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/descriptor/RedisValidator.java:33]: Could we follow the same pattern for the variables (e.g. `REDIS_CLUSTER_PASSWORD`)

File: flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/common/RedisHandlerTest.java [hunk 5]
```diff
@@ -17,13 +17,8 @@
...
 
-import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.REDIS_CLUSTER;
-import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.REDIS_COMMAND;
-import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.REDIS_KEY_TTL;
-import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.REDIS_MODE;
-import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.REDIS_NODES;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.apache.flink.streaming.connectors.redis.descriptor.RedisValidator.*;
+import static org.junit.Assert.*;
```
Comment (lresende) [inline flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/common/RedisHandlerTest.java:21]: Could we please continue using explicit imports?

**Comment by ahern88** [comment]
Ok, I have optimized it out, Please help to check, thanks

File: flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/config/handler/FlinkJedisClusterConfigHandler.java [hunk 6]
```diff
@@ -44,8 +45,16 @@ public FlinkJedisConfigBase createFlinkJedisConfig(Map<String, String> propertie
             String[] arr = r.split(":");
             return new InetSocketAddress(arr[0].trim(), Integer.parseInt(arr[1].trim()));
         }).collect(Collectors.toSet());
-        return new FlinkJedisClusterConfig.Builder()
-                .setNodes(nodes).build();
+        String clusterPassword = properties.getOrDefault(REDIS_CLUSTER_PASSWORD, null);
+        if (clusterPassword != null && clusterPassword.trim().isEmpty()) {
```
Comment (eskabetxe) [inline flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/config/handler/FlinkJedisClusterConfigHandler.java:49]: this if could be merged with the if on line 54 no?

if (clusterPassword != null && clusterPassword.trim().isEmpty()) {
       builder.setPassword(clusterPassword);
}

or directly
if (StringUtils.isNotBlank(clusterPassword))

Comment (ahern88) [reply flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/config/handler/FlinkJedisClusterConfigHandler.java]: Thanks,  i optimized the code

Comment (ahern88) [reply flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/config/handler/FlinkJedisClusterConfigHandler.java]: Thanks,  i optimized the code
if (StringUtils.isNotBlank(clusterPassword)) {
builder.setPassword(clusterPassword);
}

**Pull request merged**